loops-conditionals/conditionals/exercise_01.js

```diff
@@ -7,3 +7,15 @@
 
     HINT: The modulus operator ( % ) is your friend.
 */
+
+// 1
+
+let number = 8;
+
+// 2
+
+if (number % 2 === 0) {
```

Comment:
Good solution. 

Take in account object calisthenics rules in the future 👉 avoid `else` statement.

https://medium.com/@davidsen/clean-code-object-calisthenics-f6f4dec07c8b